solutions/USACO jan18-billboard.mdx

```diff
@@ -2,12 +2,57 @@
 id: usaco-759
 source: USACO Bronze 2018 January
 title: Blocked Billboard II
-author: Melody Yu
+author: Melody Yu, Danh Ta Chi Thanh
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html)
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html)
 
 <LanguageSection>
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+#include <iostream>
+#include <cstdio>
+using namespace std;
+
+void solve(){
+    int x[5],y[5];
+	for (int i=1;i<=4;++i){
+		cin >> x[i] >> y[i];
+	}
+	if (x[4]>=x[2] && x[3]<=x[1] && y[4]>=y[2] && y[3]<=y[1]){
+		cout << 0;
+	}
+	else if (x[3]<=x[1] && y[3]<=y[1] && y[4]>y[1] && x[4]>=x[2]){
```

Comment:
A comment for every case would be great.

Reply:
I will add them later

Reply:
The cases are difficult to explain, so I think I'll add some images to visualize rather than using words.

Reply:
Is imgur a good source to upload pictures, I'm worried that they will be removed :(((
https://imgur.com/a/OIxiZLM